cms/src/main/resources/META-INF/hst-assembly/addon/crisp/overrides/custom-resource-resolvers.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.1.xsd">
+
+    <bean id="apigeeManagementApi"
+          parent="abstractCrispSimpleJacksonRestTemplateResourceResolver"
+          class="org.onehippo.cms7.crisp.core.resource.jackson.SimpleJacksonRestTemplateResourceResolver">
+        <property name="cacheEnabled" value="false"/>
+        <property name="restTemplate">
+            <bean class="org.springframework.security.oauth2.client.OAuth2RestTemplate">
+                <constructor-arg>
+                    <bean class="org.springframework.security.oauth2.client.token.grant.password.ResourceOwnerPasswordResourceDetails">
+                        <property name="clientAuthenticationScheme" value="form"/>
+                        <property name="grantType" value="password"/>
+                        <property name="accessTokenUri" value="${devzone.apigee.oauth.token.url}"/>
+                        <property name="username" value="${DEVZONE_APIGEE_OAUTH_USERNAME}"/>
```

Comment:
Are the new environment configurations documented?

Reply:
They will be towards the end of the epic's implementation. I have a ticket to raise about that in my todo.